product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/assemble_command.mdx

```diff
@@ -2,9 +2,9 @@
 title: 'Assembling a deployment command'
 ---
 
-For a quick installation with the aim of testing the product, see the [Quick start](../quickstart/).
+For a quick installation with the aim of testing the product, see [Quick start](/postgres_distributed_for_kubernetes/latest/quickstart/).
```

Comment:
I might have been the one to steer you wrong on this. I discussed this with @djw-m to be sure I get this right going forward: use relative paths within the same section, otherwise use the full doc path.

Reply:
Hey @ebgitelman, 
Got it, I think I was mixing up "section" with "product". 
With "section", you are referring to a "folder" within a product category?

Reply:
@ebgitelman  
I talked to @djw-m and he mentioned we should use relative links for referencing topics within the same "product" directory and use full paths for referencing topics of other products or when directing to reference material. 
Hence, I am going to roll back the link suggestions, but going to merge all other suggestions. TY and LMK if you have questions!